src/Dapr.Actors/Runtime/DefaultActorTimerManager.cs

```diff
@@ -77,5 +91,24 @@ private async ValueTask<string> SerializeReminderAsync(ActorReminder reminder)
                 reminder.Ttl);
             return await info.SerializeAsync();
         }
+
+        private async ValueTask<ActorReminder> DeserializeReminderAsync(Stream stream, ActorReminderToken token)
+        {
+            if (stream == null)
+            {
+                throw new ArgumentNullException(nameof(stream));
+            }
+            var streamContents = new byte[stream.Length];
+            stream.Read(streamContents);
+            if(Encoding.ASCII.GetString(streamContents) == "null")
```

Comment:
Would the JSON deserializer not catch this case?  In particular, can we assume `ASCII` encoding vs. `UTF8`?

Reply:
No the Json Deserializer actually checks for the values in the stream and if they are not there, it assigns a default value for some(e.g. TimeSpan,Period). Also note that, in case a reminder is not there, `stream` is not null, instead it's actually has a string "null", which is what Dapr returns when the reminder is not there. So this is required for handling the case when there is no reminder.

Reply:
Handled the case in JsonDeserializer only. That's a much better and cleaner way. Thanks!